packages/terra-image/src/terra-dev-site/test/image/ImageFluid.test.jsx

```diff
@@ -15,11 +15,11 @@ const ImageFluid = () => (
     </div>
     <p>Image width is larger than the container. Image height is smaller than the container.</p>
     <div id="height-smaller" className={cx('image-conatiner-reduced-width')}>
-      <Image src={placeholder150x150} alt="rounded" variant="rounded" isFluid />
+      <Image src={placeholder150x150} alt="rounded" variant="rounded" imageFit="cover" isFluid />
     </div>
     <p>Image width is smaller than the container. Image height is larger than the container.</p>
     <div id="width-smaller" className={cx('image-conatiner-reduced-height')}>
-      <Image src={placeholder150x150} alt="rounded" variant="rounded" isFluid />
+      <Image src={placeholder150x150} alt="rounded" variant="rounded" imageFit="cover" isFluid />
```

Comment:
we could prabably add an test here for imagFit  prop values "contain","scaledown" and "none".

@neilpfeiffer , @ryanthemanuel  your thoughts.

Reply:
wouldn't hurt to add a new test with each of the 'fit' values